utils/polus-imagej-util/classes/populate.py

```diff
@@ -113,12 +113,22 @@ def __init__(self,
         self._inputs = []
         self._output = []
         
-        # Check and update if any inputs are named "in" which conflict with 
-        # python's reserved key word
+        # Check and update if any input titles
         for input_index, input in enumerate(inputs):
+            # Check if input title will interfere with reserved python keyword
             if input[1] == 'in':
-                # Change the input name from "in" to "in1"
+                # Change the input name from "in" to "in1" 
                 inputs[input_index] = (input[0], 'in1')
+            
+            # Change the name of the out as input argument so it does not 
+            # interfere with output from op
+            elif input[1] == 'out':
+                # Change name from "out" to "out_input"
+                inputs[input_index] = (input[0], 'out_input')
+
+            elif input[1] == 'out?':
+                # Change name from "out" to "out_input"
+                inputs[input_index] = (input[0], 'out_input?')
```

Comment:
When checking existence, is there a need to be more robust?

For example, do we know that `out` will always be the second input argument?

For robustness, should we do the following?
```python
if "in" in inputs:
    # Find where "in" is and change it
if "out" in inputs:
    # Find where "out" is and change it

#etc...
```